regtest/cluster.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+var path = require('path');
+var async = require('async');
+var spawn = require('child_process').spawn;
+
+var BitcoinRPC = require('bitcoind-rpc');
+var rimraf = require('rimraf');
+var bitcore = require('bitcore-lib');
+var chai = require('chai');
+var should = chai.should();
+
+var index = require('..');
+var log = index.log;
+log.debug = function() {};
+var BitcoreNode = index.Node;
+var BitcoinService = index.services.Bitcoin;
+
+describe('Bitcoin Cluster', function() {
+  var node;
+  var daemons = [];
+  var execPath = path.resolve(__dirname, '../bin/bitcoind');
+  var nodesConf = [
+    {
+      datadir: path.resolve(__dirname, './data/node1'),
+      conf: path.resolve(__dirname, './data/node1/bitcoin.conf'),
+      rpcuser: 'bitcoin',
+      rpcpassword: 'local321',
+      rpcport: 30521,
+      zmqpubrawtx: 'tcp://127.0.0.1:30611',
+      zmqpubhashblock: 'tcp://127.0.0.1:30611'
+    },
+    {
+      datadir: path.resolve(__dirname, './data/node2'),
+      conf: path.resolve(__dirname, './data/node2/bitcoin.conf'),
+      rpcuser: 'bitcoin',
+      rpcpassword: 'local321',
+      rpcport: 30522,
+      zmqpubrawtx: 'tcp://127.0.0.1:30622',
+      zmqpubhashblock: 'tcp://127.0.0.1:30622'
+    },
+    {
+      datadir: path.resolve(__dirname, './data/node3'),
+      conf: path.resolve(__dirname, './data/node3/bitcoin.conf'),
+      rpcuser: 'bitcoin',
+      rpcpassword: 'local321',
+      rpcport: 30523,
+      zmqpubrawtx: 'tcp://127.0.0.1:30633',
+      zmqpubhashblock: 'tcp://127.0.0.1:30633'
+    }
+  ];
+
+  before(function(done) {
+    log.info('Starting 3 bitcoind daemons');
+    this.timeout(20000);
```

Comment:
Not getting that error here, however I've noticed that travis ci will sometimes timeout.